fails to build with text-2.1.2

For Stackage Nightly with ghc-9.10.2:

```
       Building test suite 'test' for csv-conduit-1.0.1.0...                                                  12:58:25 [5576/954403]
       [1 of 1] Compiling Main
       /home/curators/work/unpack-dir/unpacked/csv-conduit-1.0.1.0-640699470ec7cad9c6a0a14a9764e6213556029c5c01f97c822bf035f3ceb75b/
test/Test.hs:46:34: error: [GHC-87543]
           Ambiguous occurrence ‘show’.
           It could refer to
              either ‘Prelude.show’,
                     imported from ‘Prelude’ at test/Test.hs:5:8-11
                     (and originally defined in ‘ghc-internal-9.1002.0:GHC.Internal.Show’),
                  or ‘Data.Text.show’,
                     imported from ‘Data.Text’ at test/Test.hs:16:1-16.
          |
       46 |         Left e -> assertFailure (show e)
          |                                  ^^^^

        
       /home/curators/work/unpack-dir/unpacked/csv-conduit-1.0.1.0-640699470ec7cad9c6a0a14a9764e6213556029c5c01f97c822bf035f3ceb75b/
test/Test.hs:51:34: error: [GHC-87543]
           Ambiguous occurrence ‘show’.
           It could refer to
              either ‘Prelude.show’,
                     imported from ‘Prelude’ at test/Test.hs:5:8-11
                     (and originally defined in ‘ghc-internal-9.1002.0:GHC.Internal.Show’),
                  or ‘Data.Text.show’,
                     imported from ‘Data.Text’ at test/Test.hs:16:1-16.
          |
       51 |         Left e -> assertFailure (show e)
          |                                  ^^^^
```

--- a/test/Test.hs
+++ b/test/Test.hs
@@ -13,7 +13,7 @@ import Data.CSV.Conduit.Types
 import qualified Data.Map as Map
 import qualified Data.Map.Ordered as OMap
 import Data.Monoid as M
-import Data.Text
+import Data.Text (Text)
 import qualified Data.Vector as V
 import System.Directory
 import Test.Framework (Test, defaultMain, testGroup)

